javaparser-core-testing/src/test/java/com/github/javaparser/ast/comments/JavadocTest.java

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright (C) 2007-2010 Júlio Vilmar Gesser.
+ * Copyright (C) 2011, 2013-2019 The JavaParser Team.
+ *
+ * This file is part of JavaParser.
+ *
+ * JavaParser can be used either under the terms of
+ * a) the GNU Lesser General Public License as published by
+ *     the Free Software Foundation, either version 3 of the License, or
+ *     (at your option) any later version.
+ * b) the terms of the Apache License
+ *
+ * You should have received a copy of both licenses in LICENCE.LGPL and
+ * LICENCE.APACHE. Please refer to those files for details.
+ *
+ * JavaParser is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ */
+package com.github.javaparser.ast.comments;
+
+import com.github.javaparser.ast.CompilationUnit;
+import com.github.javaparser.ast.NodeList;
+import com.github.javaparser.ast.body.MethodDeclaration;
+import com.github.javaparser.ast.comments.JavadocDescriptionElement;
+import org.junit.jupiter.api.Test;
+
+import static com.github.javaparser.StaticJavaParser.parse;
+import static com.github.javaparser.StaticJavaParser.parseMethodDeclaration;
+import static com.github.javaparser.utils.Utils.EOL;
+import com.github.javaparser.printer.PrettyPrinterConfiguration;
+
+import java.util.List;
+
+import static com.github.javaparser.utils.TestUtils.assertEqualsNoEol;
+import static java.util.stream.Collectors.toList;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+/**
+ * Tests for JavaDoc parsing.
+ */
+class JavadocTest { 
+
+    private static final PrettyPrinterConfiguration PRETTY_PRINTER_CONFIG_TWO_INDENT = new PrettyPrinterConfiguration().setIndentSize(2);
+
+    @Test
+    void toTextForEmptyJavadoc() {
+        // REQ 2
+        JavadocContent doc = new JavadocContent();
+        assertEquals("", doc.toText());
+    }
+
+    @Test
+    void toTextForJavadocWithTwoLinesOfJustDescription() {
+        // REQ 2
```

Comment:
What do all these REQ's refer to?

Reply:
The assignment said that we had to specify which tests belonged to which requirements, so we added references to the requirements as comments for presentation purposes. Sorry, we must have forgotten to remove them for the PR :(

Reply:
Okay, now I know that these can be removed